deploy/utils/backup_script/run_backup.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+#This script may be added to crontab. Example usage:
+# 0 * * * * /sportchain/backup_script/run_backup.sh <node_URL> <path_to_gcloud> <path_to_gcp_key_file> <S3-bucket name> >/dev/null 2>&1
+
+set -e
+
+LOCKFILE=/tmp/backup_script.lock
+
+#The script will not start executing until the execution of the previous script is finished
+if [ -e "$LOCKFILE" ]; then
+    echo "Script is running now. Lockfile enabled."
+    exit 1
+fi
+
+touch $LOCKFILE
+
+#Check whether the CMD arguments were passed
+if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ] || [ -z "$4" ]; then
+  echo "Error. Usage: $0 <node_URL> <path_to_gcloud> <path_to_gcp_key_file> <S3-bucket name>"
+  exit 1
+fi
+
+node_URL=$1
+gcloud_path=$2
+gcp_key_file=$3
+bucket_name=$4
+
+response=$(curl -s -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "system_health", "params":[]}' "$node_URL")
+
+# Checking the success of the response from node
+if [ -n "$response" ]; then
+  echo "The response from node has been received. Node status:"
+  echo "$response" | jq
+else
+  echo "Error. No response was received from node"
+  exit 1
+fi
+
+export PATH=$PATH:$gcloud_path
```

Comment:
This behavior can be optional.
If we are using absolute path of a binary as argument, then we should avoid to use a binary with the same name from `$PATH` (if it exists).
This way we can check if this binary is available as `command` and configure `$PATH` env variable outside the script if necessary.